herring/puzzles/tasks.py

```diff
@@ -72,10 +72,10 @@ def apply_async(*args, **kwargs):
     return t
 
 
-def post_local_and_global(local_channel, local_message, global_message):
-    logging.warning("tasks: post_local_and_global(%s, %s, %s)", local_channel, local_message, global_message)
+def post_local_and_global(local_channel, local_message, global_message, local_reaction=None, global_reaction=None):
+    logging.warning("tasks: post_local_and_global(%s, %s, %s, %s)", local_channel, local_message, global_message, local_reaction, global_reaction)
```

Comment:
You missed a `%s` there.

Reply:
Blast, I originally only included one reaction and added the 2nd one later when I realized that I was sometimes going to want it in global but not local. Will fix.